ui/components/app/currency-input/currency-input.js

```diff
@@ -32,15 +32,19 @@ import { useIsOriginalNativeTokenSymbol } from '../../../hooks/useIsOriginalNati
  * @param options0.onChange
  * @param options0.onPreferenceToggle
  * @param options0.swapIcon
+ * @param options0.isLongSymbol
  * @param options0.className
+ * @param options0.tokenSymbol
  */
 export default function CurrencyInput({
   hexValue,
   featureSecondary,
   onChange,
   onPreferenceToggle,
   swapIcon,
+  isLongSymbol = false,
```

Comment:
This can be directly calculated within the component using `tokenSymbol` and a constant so passing it as a prop is unnecessary

Also, should `primarySuffix` be used to determine `isLongSymbol` instead of just `tokenSymbol`?

Reply:
Agreed on both points